sys/usb/Kconfig

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2019 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_USB
+    bool "Configure USB"
+    depends on MODULE_USBUS
+    help
+        Configure the USB peripheral via Kconfig.
+
+if KCONFIG_USB
+
+menu "Power management"
+
+config USB_MAX_POWER
+    int "Maximum power draw [mA]"
+    default 100
+    range 0 500
+
+config USB_SELF_POWERED
+    bool "Self-powered device"
+
+endmenu # Power management
+
+config USB_PID
+    hex "Product ID"
+    range 0x0000 0xFFFF
+    help
+        You must provide your own PID.
+
+config USB_VID
+    hex "Vendor ID"
+    range 0x0000 0xFFFF
+    help
+        You must provide your own VID.
+
+config USB_MANUF_STR
+    string "Manufacturer string"
+    default "RIOT-os.org"
+
+config USB_PRODUCT_STR
+    string "Product string"
+    default "USB device"
+
+config USB_CONFIGURATION_STR
+    string "Configuration string"
+    default "USB config"
+
+config USB_PRODUCT_BCDVERSION
+    hex "Device version"
+    default 0x0100
+    help
+        This is the version number of this peripheral. Not to be confused with
+        the USB version number.
+
+config USB_SPEC_BCDVERSION
+    hex "Specification version"
+    default 0x0200
```

Comment:
I added a choice for this. Now, I'm not completely sure if the translation between the selected option and the number should be handled here or in the header file. Thoughts?

Reply:
In an offline discussion with @leandrolanzieri we figured that it might be better to keep the kconfig files lean and put the logic into the header files.

Reply:
Makes sense to me. What does this mean in practice here? I think it makes sense to have a radio-button style option for this but I don't really care if that means it selects an abstract value or the internal values :)

Reply:
Changed to assign the particular version value in the header file, keeping only a `choice` in the Kconfig file.